readme.md

```diff
@@ -18,47 +18,47 @@ CanJS is a collection of the following client-side JavaScript architectural libr
 
 _The best, most hardened and generally useful libraries in CanJS._
 
-- Constructors - [can-construct](/canjs/can-construct)
-- Observables - [can-define](/canjs/can-define) and [can-compute](/canjs/can-compute)
-- Data connection and service modeling- [can-connect](/canjs/can-connect) and [can-set](/canjs/can-set)
-- Routing - [can-route](/canjs/can-route) and [can-route-pushstate](/canjs/can-route-pushstate)
-- Live binding templates - [can-stache](/canjs/can-stache)
-- Custom elements and view bindings - [can-component](/canjs/can-component) and [can-stache-bindings](can-stache-bindings)
+- Constructors - [can-construct](https://v3.canjs.com/doc/can-construct.html)
```

Comment:
The `v3.` should probably be removed from here.  Also, these links should match what is in each collection:

 
<img width="238" alt="canjs_-_core" src="https://cloud.githubusercontent.com/assets/78602/25017598/fe07fe44-2049-11e7-9b3e-d0afae9935e9.png">